tests/jax/components/building/parameter_client_test.py

```diff
@@ -0,0 +1,238 @@
+# python3
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for parameter client components"""
+
+from types import SimpleNamespace
+
+import pytest
+
+from mava.components.jax.building.parameter_client import (
+    ExecutorParameterClient,
```

Comment:
Missing `BaseParameterClient` test

Reply:
I initially left it out since it is only there to create count parameters, but I have now included an extra mock class and test case for it. Better to test than to not test.